src/sentry/sentry_apps/services/app_request/service.py

```diff
@@ -0,0 +1,36 @@
+import abc
+
+from sentry.hybridcloud.rpc.resolvers import ByRegionName
+from sentry.hybridcloud.rpc.service import RpcService, regional_rpc_method
+from sentry.sentry_apps.services.app_request.model import (
+    RpcSentryAppRequest,
+    SentryAppRequestFilterArgs,
+)
+from sentry.silo.base import SiloMode
+
+
+class SentryAppRequestService(RpcService):
+    key = "region_app"
```

Comment:
```suggestion
    key = "sentry_app_request"
```

This name is used in metrics and URLs. Having it line up with the classname is an informal convention.